cypress/e2e/Album.cy.js

```diff
@@ -0,0 +1,70 @@
+describe('AlbumDetails', () => {
+  before(() => {
+    cy.intercept(
+      'GET',
+      'https://api.discogs.com/masters/25976?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ',
+      {
+        statusCode: 200,
+        body: {
+          "title": "Clicked Album Title",
+          "artist": "Clicked Artist",
+          "releaseDate": "2023-07-01",
+          "genre": "Pop",
+          "styles": ["Style1", "Style2"],
```

Comment:
definitely running into some albums that don't have the `styles` key - so let's conditionally render this one as well

Reply:
should i do this within the Cypress testing for this?

Reply:
whoops no, pls make that change within `Album.js` - I'll make another comment

Reply:
refactored